examples/bicluster/plot_bicluster_newsgroups.py

```diff
@@ -149,12 +139,11 @@ def most_common(d):
         continue
 
     # categories
-    counter = defaultdict(int)
+    counter = Counter()
```

Comment:
I think you can even do it without the loop:
```py
counter = Counter(document_names[i] for i in cluster_docs)
```

Reply:
I did not see this comment prior to writing mine :)

Reply:
Oh yes, that's even better.